src/views/SearchView.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="search-page" class="container">
+    <section class="search-wrapper">
+      <search-bar @query="searchPokemon" />
+    </section>
+    <section
+      v-if="(!loading && pokemon) || (!loading && error)"
+      class="tab-item"
+    >
+      <tab-item
```

Comment:
Again, I see you are planning to create some super generic component here, but as long as you don't need it, it just does not worth the headache.